packages/xo-server/src/api/vtpm.mjs

```diff
@@ -0,0 +1,27 @@
+export function create({ vm }) {
+  return this.getXapi(vm).VTPM_create(vm._xapiRef)
+}
+
+create.description = 'create a VTPM'
+
+create.params = {
+  id: { type: 'string' },
+}
+
+create.resolve = {
+  vm: ['id', 'VM', 'administrate'],
+}
+
+export function destroy({ vtpm }) {
+  return this.getXapi(vtpm).VTPM_destroy(vtpm._xapiRef)
```

Comment:
maybe it is a question for @julien-f  , what is the difference between `_xapiRef` and `$ref` ?

Reply:
`$ref` is available on all XAPI records returned by the lib `xen-api` and `_xapiRef` is available on all the XO objects after translation.

The usage is correct here :slightly_smiling_face:

Reply:
Also @MathieuRA could have used:

```suggestion
  return this.getXapiObject(vtpm).$destroy()
```